### ISSUE TITLE
Straight line [Proposal]

### ISSUE THREAD

**Issue by vvrein** [jtheoof/swappy#59]
Hi! Thank you for great tool!
Would be great if there would be possibility to draw only straight lines, eg shift + Brush draws only horizontal or vertical line as you move the pointer.

**Comment by jtheoof** [comment]
Yes indeed, I was thinking this request would come at some point.

**Comment by cmprmsd** [comment]
I'm also interested in such a feature. Also persistent line width between screenshots would assist the usability 👌

**Comment by asa1** [comment]
Adding my support for this old feature request. Thanks for swappy, it's great!

**Comment by Zoltan-Balazs** [comment]
I would also like to see this implemented, it's one of the only things that I miss.
Perhaps arrows could be used as a base, since that already does draw lines.
I could try to make a PR.
